arch/arm/boot/dts/overlays/ov5647-overlay.dts

```diff
@@ -83,4 +83,8 @@
 			};
```

Comment:
Is there a reason not to explicitly set the default rotation to 0, if only to make it obvious?